themes/ddbasic/sass/module/_ding_nodelist.scss

```diff
@@ -0,0 +1,1101 @@
+/* Colors variables
+   ========================================================================== */
+
+/* Widgets
+   ========================================================================== */
+
+.ding_nodelist-simple,
+.ding_nodelist-simple > div {
+  float: left;
+}
+
+.ding_nodelist-carousel {
+  height: 200px;
+}
+
+/* Layout
+   ========================================================================== */
+
+.ding_nodelist-col-first {
+  clear: left;
+}
+
+.ding_nodelist-col-full {
+  width: 100%;
+}
+
+.ding_nodelist-col-half {
+  width: 48%;
+  margin-right: 4%;
+}
+
+.ding_nodelist-col-third {
+  width: 31%;
+  margin-right: 3%;
+}
+
+.ding_nodelist-col-quarter {
+  width: 22%;
+  margin-right: 3%;
+}
+
+.ding_nodelist-col-last {
+  margin-right: 0;
+}
+
+.ding_nodelist-item {
+  margin-bottom: 8px;
+  margin-top: 10px;
+}
+
+/* Ding nodelist carousel
+   ========================================================================== */
+
+.ding_nodelist-carousel {
+  .item {
+    height: 170px;
+  }
+
+  .pagination {
+    position: absolute;
+  }
+
+  .next-prev{
+    position: absolute;
+  }
+}
+
+.ding_editorial_article {
+  border: 1px solid #e3e2d8;
+  background: #f9f8ef;
+  padding: 10px 15px;
+  margin-bottom: 20px;
+  text-align: justify;
+  -webkit-border-radius: 15px;
+  -moz-border-radius: 15px;
+  border-radius: 15px;
+  -webkit-box-shadow: 3px 7px 9px -7px #000000;
+  -moz-box-shadow: 3px 7px 7px -7px #000000;
+  box-shadow: 3px 7px 7px -7px #000000;
```

Comment:
It seems like we could use compass mixins for [`border-radius`](http://compass-style.org/reference/compass/css3/border_radius/) and [`box-shadow`](http://compass-style.org/reference/compass/css3/box_shadow/) here.